### DISCUSSION TITLE
Package install in TF environment

### DISCUSSION THREAD

**Discussion by mtannaan** [PaddlePaddle/PaddleSpatial#198] [BaiduKDDCup2022] [answered]
Hi organizers,

Thanks for hosting this exciting competition, and for your super-hard work in these discussions :-)
Could you install tsfresh in TensorFlow environment?

Thanks!

**Comment by ramber1836** [comment, marked as answer]
Working on it.

**Reply by ramber1836** [reply]
This package had been installed.
